vhdbuilder/packer/components.json

```diff
@@ -255,29 +256,37 @@
       "multiArchVersions": [
         "v1.26.9",
         "v1.28.6",
+        "v1.28.6-1",
         "v1.29.3",
+        "v1.29.3-1",
         "v1.29.4"
       ]
     },
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes-csi/azurefile-csi:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v1.26.11",
+        "v1.26.11-2",
+        "v1.26.11-3",
         "v1.28.9",
+        "v1.28.9-1",
         "v1.29.3",
+        "v1.29.3-1",
```

Comment:
remove this since there is already v1.29.4

Reply:
updated, thanks!